Lib/csv.py

```diff
@@ -1,6 +1,66 @@
 
 """
```

Comment:
We need to make this a raw string as otherwise `help(csv)` will show:

```
[...snip...]
DESCRIPTION
[...snip...]
       class excel:
            delimiter = ','
            quotechar = '"'
            escapechar = None
            doublequote = True
            skipinitialspace = False
            lineterminator = '
    '
            quoting = QUOTE_MINIMAL
```

```suggestion
r"""
```

Reply:
Thanks!